crates/copilot/src/copilot_chat.rs

```diff
@@ -424,12 +423,14 @@ pub fn copilot_chat_config_dir() -> &'static PathBuf {
     static COPILOT_CHAT_CONFIG_DIR: OnceLock<PathBuf> = OnceLock::new();
 
     COPILOT_CHAT_CONFIG_DIR.get_or_init(|| {
-        if cfg!(target_os = "windows") {
-            home_dir().join("AppData").join("Local")
+        let config_dir = if cfg!(any(target_os = "linux", target_os = "freebsd")) {
+            std::env::var("FLATPAK_XDG_CONFIG_HOME")
```

Comment:
I think you need to use `XDG_CONFIG_HOME`.

```
flatpak run --command=sh dev.zed.Zed
[📦 dev.zed.Zed ~]$ echo $XDG_CONFIG_HOME
/home/mona/.var/app/dev.zed.Zed/config
```

Reply:
It should fallback to that if this env is not set. dir::config_path takes care of it.

Reply:
Got it. I just made sure it's in inline with zed config: https://github.com/zed-industries/zed/blob/main/crates/paths/src/paths.rs#L75 resolution. Not sure why it exists or reason behind it for case of zed. But yes eventually it fallbacks to $XDG_CONFIG_HOME(which is handled by `dirs::config_dir()`) if it doesn't find FLATPAK_XDG_CONFIG_HOME env. So for now it should safely resolve and work. Will wait for some from zed to weigh in on whether or not we need.

Reply:
@vixalien this was introduced way before I revamped  paths see e.g. https://github.com/zed-industries/zed/commit/81475ac4cde2d7481d620ef22e1d9c3148b38ec2